contracts/bridge/README.md

```diff
@@ -712,24 +717,24 @@ This function receive a struct `multi_store_proof` as an input and then return t
 [0]   [1]  [2]   [3] [4]   [5]  [6]    [7]  [8]    [9]  [A]    [B]  [C]    [D]  [E]   [F]
 
 # Leafs of AppHash tree
-[0] - auth     [1] - authz    [2] - bank    [3] - capability [4] - crisis  [5] - dist
-[6] - evidence [7] - feegrant [8] - gov     [9] - ibccore    [A] - mint    [B] - oracle
-[C] - params   [D] - slashing [E] - staking [F] - transfer   [G] - upgrade
+[0] - acc (auth) [1] - authz    [2] - bank     [3] - capability [4] - crisis   [5] - dist
+[6] - evidence   [7] - feegrant [8] - gov      [9] - ibccore    [A] - icahost  [B] - mint
+[C] - oracle     [D] - params   [E] - slashing [F] - staking    [G] - transfer [H] - upgrade
 ```
 
 Firstly, calculate double sha256 of multi_store_proof.oracle_iavl_state_hash and then prepend with oracle prefix (uint8(6) + "oracle" + uint8(32)) and then calculate merkle_leaf_hash
 
-1. **_[B]_** = [merkle_leaf_hash](#merkle_leaf_hash)(0x066f7261636c6520 + [sha256](#utility-functions)([sha256](#utility-functions)(`oracleIAVLStateHash`)))
+1. **_[D]_** = [merkle_leaf_hash](#merkle_leaf_hash)(0x066f7261636c6520 + [sha256](#utility-functions)([sha256](#utility-functions)(`oracleIAVLStateHash`)))
 
-2. **_[I5]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[A]_**, **_[B]_**)
+2. **_[I7]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[E]_**, **_[F]_**)
 
 3. **_[I10]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[I4]_**, **_[I5]_**)
 
-4. **_[I13]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[I10]_**, **_[I11]_**)
+4. **_[I12]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[I8]_**, **_[I9]_**)
 
-5. **_[I14]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[I12]_**, **_[I13]_**)
+5. **_[I15]_** = [merkle_inner_hash](#merkle_inner_hash)(**_[G]_**, **_[H]_**)
```

Comment:
seem wrong?

Reply:
please check this. https://github.com/bandprotocol/vrf-and-bridge-contracts/pull/4